packages/core/src/domain/telemetry/telemetry.spec.ts

```diff
@@ -48,41 +48,21 @@ describe('telemetry', () => {
       resetExperimentalFeatures()
     })
 
-    it('should collects configuration when sampled and ff is enabled', () => {
-      updateExperimentalFeatures(['telemetry_configuration'])
-
+    it('should collects configuration when sampled', () => {
       const { notifySpy } = startAndSpyTelemetry({ telemetrySampleRate: 100, telemetryConfigurationSampleRate: 100 })
 
       addTelemetryConfiguration({})
 
       expect(notifySpy).toHaveBeenCalled()
     })
 
-    it('should not notify configuration when sampled and ff is disabled', () => {
-      const { notifySpy } = startAndSpyTelemetry({ telemetrySampleRate: 100, telemetryConfigurationSampleRate: 100 })
-
-      addTelemetryConfiguration({})
-
-      expect(notifySpy).not.toHaveBeenCalled()
-    })
-
-    it('should not notify configuration when not sampled and ff is enabled', () => {
-      updateExperimentalFeatures(['telemetry_configuration'])
+    it('should not notify configuration when not sampled', () => {
       const { notifySpy } = startAndSpyTelemetry({ telemetrySampleRate: 100, telemetryConfigurationSampleRate: 0 })
 
       addTelemetryConfiguration({})
 
       expect(notifySpy).not.toHaveBeenCalled()
     })
-
-    it('should not notify configuration when telemetrySampleRate is 0 and ff is enabled', () => {
-      updateExperimentalFeatures(['telemetry_configuration'])
-      const { notifySpy } = startAndSpyTelemetry({ telemetrySampleRate: 0, telemetryConfigurationSampleRate: 100 })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should not we keep this test?

Reply:
true